source/code/source/playfab/QoS/PlayFabQoSApi.cpp

```diff
@@ -387,7 +387,7 @@ namespace PlayFab
         // Parameters : Configured socket to ping
         // Return : The ping result
         // Note that the function eat any exceptions thrown to it.
-        PingResult PlayFabQoSApi::GetQoSResultForRegion(shared_ptr<QoSSocket> socket)
+        PingResult PlayFabQoSApi::GetQoSResultForRegion(const shared_ptr<QoSSocket>& socket)
```

Comment:
Since the function below uses a non const labeled function, it does make sense to not make this const. That function does utilize unique member functions that may be "modified"